### DISCUSSION TITLE
How to invoke Telegram methods?

### DISCUSSION THREAD

**Discussion by GRUDOVOFF** [def-studio/telegraph#500] [Q&A] [answered]
I have recently started working with this library and Laravel in general. Please show me an example of getchatmember method and how to call it and get data.

**Comment by fabio-ivona** [comment]
hi, please refer to the documentation site: https://defstudio.github.io/telegraph/features/telegram-api-calls#chatmember

**Reply by GRUDOVOFF** [reply]
Thanks, but I can't figure out how to use this in code. I need to make a check for subscribing to a Telegram channel. Please write me a piece of code with this method.

**Reply by fabio-ivona** [reply, marked as answer]
You need to have a user's Telegram $userId in order to retrieve its chat member info: 

```php
$response = Telegraph::chatMember($userId)->send();

$response->dump();
```

and then use data in your logic

can't help further without a context
